### PR TITLE
Update links to 0.7.3

### PR DESCRIPTION
**Pull request by ctlaltdieliet**
Modified the download links to 0.7.3

### CODE REVIEW

File: website/site/content/download/personal-edition/ubuntu-upgrade.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ Use the URL of the Ubuntu archive package, `focalboard-server-linux-amd64.tar.gz
 Create and use a clean directory, or delete any existing packages first, then run:
 
 ```
-# Download the new version (e.g. 0.7.0 here, check the release for the latest one)
-wget https://github.com/mattermost/focalboard/releases/download/v0.7.0/focalboard-server-linux-amd64.tar.gz
+# Download the new version (e.g. 0.7.3 here, check the release for the latest one)
```
Comment (chenilim) [inline website/site/content/download/personal-edition/ubuntu-upgrade.md:15]: Thanks for staying on top of this @ctlaltdieliet! :) However, since 0.7.3 is a plugin-only release, we should not update the Personal Server download doc - there's no download for Personal Server.

So, please revert the change un ubuntu-upgrade.md, thanks.

**Review by chenilim — CHANGES_REQUESTED** [review]
Thanks @ctlaltdieliet! Just need the change in latest-plugin/_index.md.

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:

**Review by chenilim — APPROVED** [review]
Thanks @ctlaltdieliet!

**Review by ogi-m — APPROVED** [review]
